frontend/packages/console-app/src/components/nodes/NodesPage.tsx

```diff
@@ -29,83 +40,203 @@ import { nodeStatus } from '../../status/node';
 import NodeRoles from './NodeRoles';
 import { menuActions } from './menu-actions';
 import NodeStatus from './NodeStatus';
+import { RootState } from '@console/internal/redux';
 
-const tableColumnClasses = [
-  '',
-  '',
-  '',
-  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
-  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
-  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
-  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
-  classNames('pf-m-hidden', 'pf-m-visible-on-lg'),
-  Kebab.columnClass,
-];
+const nodeColumnInfo = Object.freeze({
+  name: {
+    classes: '',
+    id: 'name',
+    title: 'Name',
+  },
+  status: {
+    classes: '',
+    id: 'status',
+    title: 'Status',
+  },
+  role: {
+    classes: '',
+    id: 'role',
+    title: 'Role',
+  },
+  pods: {
+    classes: classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+    id: 'pods',
+    title: 'Pods',
+  },
+  memory: {
+    classes: classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+    id: 'memory',
+    title: 'Memory',
+  },
+  cpu: {
+    classes: classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+    id: 'cpu',
+    title: 'CPU',
+  },
+  filesystem: {
+    classes: classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+    id: 'filesystem',
+    title: 'Filesystem',
+  },
+  created: {
+    classes: classNames('pf-m-hidden', 'pf-m-visible-on-lg'),
+    id: 'created',
+    title: 'Created',
+  },
+  instancetype: {
```

Comment:
We should use camel case IDs

```suggestion
  instanceType: {
```

Reply:
Done.